### DISCUSSION TITLE
Problem in setting up the project and one suggestion

### DISCUSSION THREAD

**Discussion by sushant4612** [ruxailab/RUXAILAB#320] [Q&A] [answered]
I followed all the steps that are present in the README file and it's running without errors or warnings however when I try to log in with the credentials I used during signup I encounter the following pop-up Initially I couldn't understand it as I am not familiar with portuguese  after searching, I found out that it's indicating incorrect email and password.

<img width="1470" alt="Screenshot 2024-02-27 at 9 48 39 PM" src="https://github.com/ruxailab/RUXAILAB/assets/109458485/8a5d7cd1-ccf2-489a-91bd-99f13f29dab3">

**suggestion** : i think the pop-up should be in english as most people couldn't get what's it showing

**Comment by jvJUCA** [comment, marked as answer]
Dear @sushant4612 

The popup translation is already fixed on the newest version of develop, we are currently working on improve the documentation to help contributors 

@jvJUCA

**Reply by sushant4612** [reply]
Okay, so until then, what should I do to learn more about the project?

**Reply by KarinePistili** [reply]
Hello @sushant4612, we've published a video guide to help setup the project correclty using Firebase Emulators. It is located on the readme. You can check out the code, create user and heuristic tests, invite collaborators and so on to get to know the code better. If you feel like you want something more academic, you can take a look at the papers we published about RUXAILAB such as [Whitepaper: Remote usability laboratory to perform user and expert evaluations
](https://dl.acm.org/doi/abs/10.1145/3612783.3612804)

**Reply by sushant4612** [reply]
@KarinePistili thanks for the video it really helps in setting up the repo locally and yes i will definitely check out the paper

**Comment by KarinePistili** [comment]
I'll be closing this discussion, as we already provided the guide video for project setup and the translation of the popup.
